frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -34,6 +38,50 @@ function FileManagerHistory:onSetDimensions(dimen)
     self.dimen = dimen
 end
 
+function FileManagerHistory:printFileInformation(book_props)
+    if book_props.authors == "" then
+        book_props.authors = _("N/A")
+    end
+
+    if book_props.title == "" then
+        book_props.title = _("N/A")
+    end
+
+    if book_props.series == "" then
+        book_props.series = _("N/A")
+    end
+
+    if book_props.pages == "" then
+        book_props.pages = _("N/A")
+    end
+
+    if book_props.language == "" then
+        book_props.language = _("N/A")
+    end
+
+    return {
+        { _("Title") .. " : " .. book_props.title, "" },
```

Comment:
This is a good usecase for the template function. See: https://github.com/koreader/koreader/blob/master/frontend/apps/reader/modules/readerfrontlight.lua#L9.

You can rewrite it as `T(_("Title: %1"), book_props.title)`. Using template makes it easier for translator to translate dynamic sentence since it provides more context. They can now translate the whole `Title: %1` sentence instead of just `Title`.